README.md

```diff
@@ -30,14 +30,18 @@ targets: '[{"project": "Backlog", "section": "Development Done"}, {"project": "C
 ```
 if you don't want to move task omit `targets`.
 
+## Sample PR Description
+``
+**Asana Task:** [Task Name](https://app.asana.com/0/1/2)
+``
 
 ## Example usage
 
 ```yaml
-uses: https://github.com/insurify/github-actions@v2.0.0
+uses: insurify/github-asana-action@v2.1.0
```

Comment:
```suggestion
uses: insurify/github-asana-action@v2.0.1
```

Reply:
@gshkolnik if i'm not wrong the last number represents the patch and I don't think we are patching something in release 2, thoughts ?

Reply:
Yes, what we're doing is a simple, backward-compatible bug fix or in other words a patch. We are certainly not doing a major release here.
I did get the version wrong though:
```suggestion
uses: insurify/github-asana-action@v1.0.1
```

Reply:
@gshkolnik the current code have one extra thing of `TARGETS` so if we add this to `1.0.1` targets will also be added to this version, so I think it's better to go with `2.0.1` or `2.1.0`.

Reply:
let's split the difference and go with `1.1.0` - sound good? :)

Reply:
Sorry, we are unable to explain it, we already have marked the last version as `2.0.0` so moving from `2.0.0` to `1.1.0` doesn't make sense.

Reply:
I thought the current version was 1.0. At least this is what we're using in ensurify repo.

Reply:
Yes you're right, we haven't updated the version in our repos.

Reply:
@gshkolnik so are we good with `2.0.1` ?

Reply:
@iamalisajid  - no, this change does not amount to a major release. My preference is that we actually don't change the version at all really. However, if you feel that we need to, we can go with `1.0.1` or maybe the highest as `1.1.0`.